Dockerfile.prod

```diff
@@ -0,0 +1,21 @@
+FROM node:12
```

Comment:
Maybe we could opt for the latest Node, 13?

Reply:
I opted for the latest LTS for stability, which is also what I'm using locally.